app/controllers/curation_concerns/generic_works_controller.rb

```diff
@@ -26,6 +35,13 @@ def identifiers
       wants.json { render :show, status: :ok, location: polymorphic_path([main_app, curation_concern]) }
     end
   end
+ 
+  def check_date_coverage
```

Comment:
Is this function actually checking something?  It appears to be filling in the date_coverage parameter under 'generic_work'.